templates/prometheus.go

```diff
@@ -109,6 +109,7 @@ var PrometheusTemplate = promv1.Prometheus{
 	},
 }
 
+//nolint:deadcode,unused
```

Comment:
- if this file is going to stay, let's remove this unused function

Reply:
Ohad asked to add it as we are going to use it in near future. I have kept it as comment and removed nolint marker.